state/state.go

```diff
@@ -67,13 +78,23 @@ func loadState(db dbm.DB, key []byte) *State {
 		}
 		// TODO: ensure that buf is completely read.
 	}
+
+	v := s.loadValidators(s.LastBlockHeight)
+	if v != nil {
+		s.lastHeightValidatorsChanged = v.LastHeightChanged
+	} else {
+		s.lastHeightValidatorsChanged = 1
```

Comment:
Ok I went back on this. height==0 isn't a thing so we should keep this at 1 for consistency. `/validators?height=0` returns an error just like `/block?height=0`